src/pages/Main/Main.js

```diff
@@ -0,0 +1,229 @@
+    imgURL: '',
+    korTitle: '',
+    engTitle: '',
+    details: '',
+    roastedDate: '',
+    price: '',
+  });
+  const [newValue, setNewValue] = useState();
+
+  // mock data 가져오기
+
+  async function request() {
+    const res = await fetch('http://localhost:3000/data/productCardBest.json');
+    const result = await res.json();
+    setValues(result);
+  }
+
+  async function requestNewData() {
+    const res = await fetch('http://localhost:3000/data/productCardNew.json');
+    const result = await res.json();
+    setNewValue(result);
+  }
+
+  useEffect(() => {
+    request();
+    requestNewData();
+  }, []);
+
+  // 이미지 슬라이드 기능 구현
+  const showPrevImage = () => {
+    imgId === 1 ? setImgId(IMAGE_DATA.length) : setImgId(prevId => prevId - 1);
+  };
+
+  const showNextImage = () => {
+    imgId === IMAGE_DATA.length ? setImgId(1) : setImgId(prevId => prevId + 1);
+  };
+
+  useEffect(() => {
+    const cycleImage = () => {
+      imgId === IMAGE_DATA.length
+        ? setImgId(1)
+        : setImgId(prevId => prevId + 1);
+    };
+
+    const autoSlide = setInterval(cycleImage, 4000);
+    return () => {
+      clearInterval(autoSlide);
+    };
+  }, [imgId]);
+
+  if (values.length) {
+    return (
+      <section className="main" id="home">
+        <div className="mainSliderContainer">
+          {IMAGE_DATA.map(image => {
+            return (
+              <Slide
+                key={image.id}
+                id={image.id}
+                url={image.url}
+                subTitle={image.subTitle}
+                Title1={image.Title1}
+                Title2={image.Title2}
+                imgId={imgId}
+              />
+            );
+          })}
+          <div className="mainSliderRightArrow" onClick={showNextImage}>
+            <i className="bx bx-chevron-right" />
+          </div>
+          <div className="mainSliderLeftArrow" onClick={showPrevImage}>
+            <i className="bx bx-chevron-left" />
+          </div>
+        </div>
+        <div className="mainContainer">
+          <div className="upperMargin">
+            <div className="mainBestList">
+              <div className="bestListTitleContainer">
+                <div className="bestListTitle">BEST</div>
+                <a href="#home" className="bestListLink">
+                  + SHOP
+                </a>
+              </div>
+              <ul className="bestListProductRecommend">
+                {values.map(product => {
+                  return (
+                    <ProductCard
+                      key={product.id}
+                      id={product.id}
+                      imgURL={product.imgURL}
+                      korTitle={product.korTitle}
+                      engTitle={product.engTitle}
+                      details={product.details}
+                      roastedDate={product.roastedDate}
+                      price={product.price}
+                      cardSize="Big"
+                    />
```

Comment:
```suggestion
                    <ProductCard
                      key={product.id}
                      cardSize="Big"
                      {...product}
                    />
```
이와 같은 방법으로도 props를 넘겨줄 수 있는데
이건 product에 해당하는 모든 값을 넘겨주겠다라는 의미입니다!